system/settings/settings.xml

```diff
@@ -1008,18 +1008,21 @@
           <default>false</default>
           <control type="toggle" />
         </setting>
-        <setting id="videolibrary.groupsingleitemsets" type="boolean" label="20470" help="36157">
+        <setting id="videolibrary.groupsingleitemsets" type="boolean" parent="videolibrary.groupmoviesets" label="20470" help="36157">
           <level>1</level>
           <default>false</default>
           <control type="toggle" />
+          <dependencies>
+            <dependency type="enable" setting="videolibrary.groupmoviesets" operator="is">true</dependency>
+          </dependencies>
         </setting>
```

Comment:
I've attempted to simplify the wording here and as part of that made the "videolibrary.groupsingleitemsets" dependant on "videolibrary.groupmoviesets"

@Montellese  Are you ok with this since you introduced the setting? want to be sure there wasn't a reason for you to not do it this way in the first place.